gtk/src/callbacks.c

```diff
@@ -4532,17 +4532,18 @@ ghb_browse_uri(signal_user_data_t *ud, const gchar *uri)
 G_MODULE_EXPORT void
 about_action_cb(GSimpleAction *action, GVariant *param, signal_user_data_t *ud)
 {
-    GtkWidget *widget = GHB_WIDGET (ud->builder, "hb_about");
+    GtkWidget *dialog = GHB_WIDGET (ud->builder, "hb_about");
```

Comment:
First of all, thanks.  Solves the problem without affecting newer gtk versions where the dialog button does not appear.

But could you please revert the change to the `widget` variable name.  It's not necessary and makes a 2 line patch into a patch that affects pretty much every line in the function.

Reply:
Sorry about that; diff does look much cleaner now. :)